cluster/gce/gci/configure-helper.sh

```diff
@@ -375,6 +375,13 @@ current-context: service-account-context
 EOF
 }
 
+function create-master-etcd-auth {
+  local -r auth_dir="/etc/srv/kubernetes"
+  echo "${ETCD_CA_CERT}" | base64 --decode > "${auth_dir}/etcd-ca.crt"
```

Comment:
this doesn't appear to be optional, so anyone (like gke) who calls this script w/o setting the new fields will fail here.